services/booking-api/babel.config.js

```diff
@@ -0,0 +1,5 @@
+const babelBase = require('../../babel-base.config');
+
+module.exports = {
+  ...babelBase,
+};
```

Comment:
What is the prupose of adding this configuration, is it in order for jest to function? :)

Reply:
Yes, its because we are using ECM